packages/mdc-form-field/_mixins.scss

```
@@ -66,4 +66,24 @@
       }
     }
   }
-}
+
+  .mdc-form-field--space-between {
+    @include feature-targeting-mixins.targets($feat-structure) {
+      justify-content: space-between;
+    }
+    // stylelint-disable-next-line selector-max-type
+    > label {
+      @include feature-targeting-mixins.targets($feat-structure) {
+        @include rtl-mixins.reflexive-property(margin, auto, initial);
+      }
+    }
+    &.mdc-form-field--align-end {
+      // stylelint-disable-next-line selector-max-type
+      > label {
+        @include feature-targeting-mixins.targets($feat-structure) {
+          @include rtl-mixins.reflexive-property(margin, initial, auto);
+        }
+      }
+    }
+  }
+}
```

Last round of changes. I realized that we don't need `auto` at all, we just need to set the margin to initial.

```suggestion
  .mdc-form-field--space-between {
    @include feature-targeting-mixins.targets($feat-structure) {
      justify-content: space-between;
    }

    // stylelint-disable-next-line selector-max-type
    > label {
      @include feature-targeting-mixins.targets($feat-structure) {
        margin: initial;

        @include rtl-mixins.rtl {
          // RTL needed for specificity
          margin: initial;
        }
      }
    }
  }
}

```